From 9c71e6c97d4d8d16a3809e4944a0fe0ff9e72849 Mon Sep 17 00:00:00 2001
From: Loic Poulain <loic.poulain@intel.com>
Date: Thu, 17 Apr 2014 03:06:39 +0200
Subject: [PATCH] 8250_dw: Remove the temporally byt DMA hack

825 DMA is now fixed to avoid any HCI hardware error.
Remove manual DMA disabling.

Change-Id: Ia655570e766be3adc78907792c94acc3ed21f24f
Signed-off-by: Loic Poulain <loic.poulain@intel.com>
---
 drivers/tty/serial/8250/8250_dw.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/tty/serial/8250/8250_dw.c b/drivers/tty/serial/8250/8250_dw.c
index 806a6f6..7d3e4d9 100644
--- a/drivers/tty/serial/8250/8250_dw.c
+++ b/drivers/tty/serial/8250/8250_dw.c
@@ -131,9 +131,6 @@ static void byt_serial_setup(struct uart_8250_port *up)
 	struct uart_port *p = &up->port;
 
 	p->set_termios = byt_set_termios;
-
-	/* WORKAROUND => Force DMA disabled */
-	up->dma = NULL;
 }
 
 static inline int dw8250_modify_msr(struct uart_port *p, int offset, int value)
-- 
1.8.3.2

